internal/backend/oauth/oauth.go

```diff
@@ -285,6 +285,25 @@ func New(l *zap.Logger) sdkservices.OAuth {
 				},
 			},
 
+			"hubspot": {
+				ClientID:     os.Getenv("HUBSPOT_CLIENT_ID"),
+				ClientSecret: os.Getenv("HUBSPOT_CLIENT_SECRET"),
```

Comment:
Is there a ticket to add these vars to staging and prod? (And store their management link and secrets in 1Password)

Reply:
See INT-8 for example